eda_housing.ipynb

```diff
@@ -0,0 +1,385 @@
+       "      <td>17.117500</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>max</th>\n",
+       "      <td>88.976200</td>\n",
+       "      <td>100.000000</td>\n",
+       "      <td>27.740000</td>\n",
+       "      <td>1.000000</td>\n",
+       "      <td>0.871000</td>\n",
+       "      <td>8.780000</td>\n",
+       "      <td>100.000000</td>\n",
+       "      <td>12.126500</td>\n",
+       "      <td>24.000000</td>\n",
+       "      <td>711.000000</td>\n",
+       "      <td>22.000000</td>\n",
+       "      <td>396.900000</td>\n",
+       "      <td>37.970000</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "             CRIM          ZN       INDUS        CHAS         NOX          RM  \\\n",
+       "count  394.000000  394.000000  394.000000  394.000000  394.000000  394.000000   \n",
+       "mean     3.690136   11.460660   11.000863    0.068528    0.553215    6.280015   \n",
+       "std      9.202423   23.954082    6.908364    0.252971    0.113112    0.697985   \n",
+       "min      0.006320    0.000000    0.460000    0.000000    0.389000    3.561000   \n",
+       "25%      0.081955    0.000000    5.130000    0.000000    0.453000    5.879250   \n",
+       "50%      0.268880    0.000000    8.560000    0.000000    0.538000    6.201500   \n",
+       "75%      3.435973   12.500000   18.100000    0.000000    0.624000    6.605500   \n",
+       "max     88.976200  100.000000   27.740000    1.000000    0.871000    8.780000   \n",
+       "\n",
+       "              AGE         DIS         RAD         TAX     PTRATIO           B  \\\n",
+       "count  394.000000  394.000000  394.000000  394.000000  394.000000  394.000000   \n",
+       "mean    68.932741    3.805268    9.403553  406.431472   18.537563  358.490939   \n",
+       "std     27.888705    2.098571    8.633451  168.312419    2.166460   89.283295   \n",
+       "min      2.900000    1.129600    1.000000  187.000000   12.600000    2.600000   \n",
+       "25%     45.475000    2.110100    4.000000  280.250000   17.400000  376.707500   \n",
+       "50%     77.700000    3.199200    5.000000  330.000000   19.100000  392.190000   \n",
+       "75%     94.250000    5.116700   24.000000  666.000000   20.200000  396.900000   \n",
+       "max    100.000000   12.126500   24.000000  711.000000   22.000000  396.900000   \n",
+       "\n",
+       "            LSTAT  \n",
+       "count  394.000000  \n",
+       "mean    12.769112  \n",
+       "std      7.308430  \n",
+       "min      1.730000  \n",
+       "25%      7.125000  \n",
+       "50%     11.300000  \n",
+       "75%     17.117500  \n",
+       "max     37.970000  "
+      ]
+     },
+     "execution_count": 86,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Exploratory data analysis on Boston housing data\n",
+    "df = load_dataset(\"housing_data.csv\")\n",
```

Comment:
Add a path variable, and please load the dataset folder on the drive in order to have same files and path.